src/Features/LanguageServer/Protocol/Handler/Completion/CompletionHandler.cs

```diff
@@ -26,10 +28,23 @@ namespace Microsoft.CodeAnalysis.LanguageServer.Handler
     [ExportLspMethod(LSP.Methods.TextDocumentCompletionName)]
     internal class CompletionHandler : AbstractRequestHandler<LSP.CompletionParams, LSP.CompletionItem[]>
     {
+        private readonly string[] _csTriggerCharacters;
+        private readonly string[] _vbTriggerCharacters;
```

Comment:
Let's make these a set so we don't have to worry about duplicates and contains is more straightforward